Tests/Testo/Invoice/Inv/InvServiceTest.php

```
@@ -0,0 +1,672 @@
+        $repo = m::mock(InvRepository::class);
+        /** @var \Mockery\Expectation $e3 */
+        $e3 = $repo->shouldReceive('save');
+        $e3->once()->with($model);
+
+        $service = $this->makeService($repo, $translator, null, $gR);
+        $service->saveInv($user, $model, [], $s, $gR);
+
+        // The translated string is computed but never assigned back onto the
+        // model (see POSSIBLE ISSUE in the final report) - terms stays at
+        // its untouched constructor default.
+        Assert::same('', $model->getTerms());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Testo\Assert' in method 'saveInvLeavesTermsUnchangedWhenArrayOmitsTermsDespiteCallingTranslator'. (undefined)

(StaticAccess)